USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -42,14 +45,17 @@ public XWPFDocument Render(USFMDocument input)
             foreach (Marker marker in input.Contents)
                 {
 
-                    RenderMarker(marker);
+                    RenderMarker(marker, new StyleConfig());
 
                 }
             return newDoc;
 
         }
-        private void RenderMarker(Marker input, XWPFParagraph parentParagraph = null, bool isBold = false, bool isItalics = false)
+        private void RenderMarker(Marker input, StyleConfig styles, XWPFParagraph parentParagraph = null, bool styleChange = false)
         {
+            StyleConfig markerStyle = styles;
```

Comment:
Actually I think this is wrong. We'll still run into the problem of overriding the parent fields because a child only knows what that its parent has a changed style but a parent doesn't know that its child is going to modify its style which is the main problem.

Reply:
I decided to make a clone of the StyleConfig on each call to RenderMarker